packages/list-react/src/List.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { UnorderedList, OrderedList, ListItem } from "../src";
+
+afterEach(cleanup);
+
+test(`List and ListItem should render as expected`, () => {
+    const { getByText } = render(
+        <UnorderedList>
+            <ListItem>Kibogiedo</ListItem>
+            <ListItem>Ovoopisow</ListItem>
+            <ListItem>Omocebig</ListItem>
+        </UnorderedList>,
+    );
+    expect(getByText("Kibogiedo")).toBeInTheDocument();
+    expect(getByText("Ovoopisow")).toBeInTheDocument();
+    expect(getByText("Omocebig")).toBeInTheDocument();
+});
```

Comment:
Lag en tilsvarende test for `OrderedList` også (typecheck feiler nå siden `OrderedList` importeres uten å brukes, og den bør uansett testes 😊).